app/controllers/wishlists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class WishlistsController < ApplicationController
```

Comment:
Similar comment as the one on `AudioFilesController`.

From what I can tell, it looks like the ApplicationController (thus this controller) , isn't requiring a user to be logged in to access it. Should it?